<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOwnerDialog</class>
 <widget class="QDialog" name="AddOwnerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLastName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFirstName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMiddleName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Пол</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxSex">
           <item>
            <property name="text">
             <string>м</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ж</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Возраст</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAge"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Телефон</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTelephon"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Количество Авто</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditCarNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Количество Жителей</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditResidenceNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Номер Дома</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxHouseNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Номер Квартиры</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAppartmentNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Тип Квартиры</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxApartmentType">
           <item>
            <property name="text">
             <string>x3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x4</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Дата прописки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Площадь квартиры</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxApartmentArea">
           <item>
            <property name="text">
             <string>97</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>132</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="btnAddOwner">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
